staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -200,10 +200,17 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string
-	// Resources is a list of resources within the API group.
-	// Any empty list implies every resource kind in the API group.
+	// Resources is a list of resources within the API group. Specific subresources
+	// must be matched using a "/" to indicate the subresource. For example "pods" does
+	// not match "pods/logs".
+	// An empty list implies every resource kind in the API group.
```

Comment:
> Do we want to add a wildcard for subresource matching? E.g. pods/* matches all pod subresources? I'm not sure whether this would be useful.

We have talked about adding this sort of `*` capability as an additive change to RBAC in support of polymorphic endpoints like `*/scale`.  Normally, `resource` *must not* allow `resource/subresource` since the subresources provide more power.  `pods/bind`, `pods/status`, `namespaces/finalize`, or any other status subresource.

@kubernetes/sig-auth-misc @liggitt